packages/date-picker/src/picker.vue

```diff
@@ -742,7 +753,7 @@ export default {
         if (!this.ranged) {
           this.handleChange();
           this.pickerVisible = this.picker.visible = false;
-          this.blur();
+          // this.blur();
```

Comment:
Why is this commented

Reply:
This allows the focus to remain on the triggering input text box and next tabbing takes the user to next in line tab index